lua/telescope/builtin/internal.lua

```diff
@@ -87,6 +87,66 @@ internal.planets = function(opts)
   }:find()
 end
 
+internal.symbols = function(opts)
+  -- TODO(conni2461): Is their a better way to get the extension path?
+  local sourced_file = require('plenary.debug_utils').sourced_filepath()
+  local base_directory = vim.fn.fnamemodify(sourced_file, ":h:h:h:h:h")
+  local extension_path = vim.fn.globpath(base_directory .. '/telescope-symbols.nvim/data/', '*', true, true)
```

Comment:
Note to myself. We could just create a function inside extension which returns the `extension_data_files` and call that function with pcall

Reply:
`vim.api.nvim_get_runtime_file('data/telescope-symbols/', false)`

Reply:
just change where it gets saved and use that API probably.

Reply:
Files should just always go in `data/telescope-symbols/*.json`

Then we can always find them easily with `vim.api.nvim_get_runtime_file()` you just add your plugin with the correct data things.